src/deploy/deploy_to_pipedream.py

```
@@ -371,30 +411,51 @@ async def wait_for_login(self) -> bool:
-            if "/workflows" in current_url or "/projects" in current_url:
-                print("\nLogin successful!")
-                return True
+                pages = [p for p in (self.context.pages if self.context else []) if not p.is_closed()]
+                if self.page and not self.page.is_closed() and self.page not in pages:
+                    pages.append(self.page)
+                for page in pages:
+                    try:
+                        if "pipedream.com" not in page.url:
+                            continue  # still inside Google's SSO flow
+                    except Exception:
+                        continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)